docs/arcadia-users-group/20221017-conda/lesson.md

```diff
@@ -0,0 +1,369 @@
+Mamba is a drop in replacement for conda and it improves installation speeds by decreasing the time it takes to solve dependency conflicts.
+You can install mamba with:
+
+```
+conda install -c conda-forge mamba
+```
+
+### Conda channels
+
+Channels are the locations of the repositories (directories) online containing Conda packages. 
+Upon Conda’s installation, Continuum’s (Conda’s developer) channels are set by default, so without any further modification, these are the locations where your Conda will start searching for packages.
+
+Channels in Conda are ordered. 
+The channel with the highest priority is the first one that Conda checks, looking for the package you asked for. 
+You can change this order, and also add channels to it (and set their priority as well).
+
+You should follow the channel order below -- otherwise, sometimes you'll encounter weird bugs because packages will be installed from the wrong location. 
+```
+conda config --add channels defaults
+conda config --add channels bioconda
+conda config --add channels conda-forge
+```
+
+Once we have our channel order configured, you will no longer need to use the explicit channel parameter `-c` when you install new packages.
+
+<center>
+<figure markdown>
+  ![condachannel](conda_channels.png){ width="700" }
+  <figcaption> Cartoon of conda channels by Gergely Szerovay <a href='https://www.freecodecamp.org/news/why-you-need-python-environments-and-how-to-manage-them-with-conda-85f155f4353c/' target='_blank'>www.freecodecamp.org</a> </figcaption>
+</figure>
+</center>
+
+If multiple channels contain a package, and one channel contains a newer version than the other one, the order of the channels determines which of these two versions is going to be installed, even if the higher priority channel contains the older version.
+
+
+<center>
+<figure markdown>
+  ![condachannels2](conda_channels2.png){ width="700" }
+  <figcaption> Cartoon of conda channel priorities by Gergely Szerovay <a href='https://www.freecodecamp.org/news/why-you-need-python-environments-and-how-to-manage-them-with-conda-85f155f4353c/' target='_blank'>www.freecodecamp.org</a> </figcaption>
+</figure>
+</center>
+
+
+### Creating a new environment
+
+Other than mamba, we recommend you keep you `base` environment free from all other installations. 
+If you install everything in `base`, you'll end up with lots of dependencies that need to be solved which will decrease installation speeds.
+It also becomes more difficult to document software installations and versions required for specific project. 
+
+To create a new environment named, for instance `mynewenv` (you can name it what ever you like), run:
+
+```
+conda create --name mynewenv
+```
+
+### Navigating environments through `activate` and `deactivate`
+
+If you want to use an environment (for instance manage packages, or run Python scripts inside it) you need to first activate it.
+
+```
+conda activate mynewenv
```

Comment:
For this to work, I think people will have to run `conda init`. Which they may or may not based on installation instructions. So I'd give a callout to that